lib/openstudio-standards/schedules/parametric.rb

```diff
@@ -510,9 +510,15 @@ def self.schedule_ruleset_get_parametric_inputs(schedule_ruleset, space_load_ins
 
       # set scheduleRuleset properties
       props = schedule_ruleset.additionalProperties
-      props.setFeature('param_sch_ver', '0.0.1') # this is needed to see if formulas are in sync with version of standards that processes them also used to flag schedule as parametric
-      props.setFeature('param_sch_floor', min_max['min'])
-      props.setFeature('param_sch_ceiling', min_max['max'])
+
+      if props.getFeatureAsString('param_sch_ver') == '0.0.1'
+        # don't need to gather more than once
+        return parametric_inputs
+      else
+        props.setFeature('param_sch_ver', '0.0.1') # this is needed to see if formulas are in sync with version of standards that processes them also used to flag schedule as parametric
+        props.setFeature('param_sch_floor', min_max['min'])
+        props.setFeature('param_sch_ceiling', min_max['max'])
+      end
```

Comment:
This just avoids the same schedule (applied to different space load instances) being 'set up' multiple times